tools/wpt/testfiles.py

```diff
@@ -199,9 +199,10 @@ def load(manifest_path=None):
 
 load_manifest = _init_manifest_cache()
 
```

Comment:
Nit: two blank lines before top-level class/function definitions is recommended (by PEP8 I think).

Reply:
Any idea why flake8 doesn't complain about this?

Reply:
Because we disabled that, because @jgraham objected to it, viewing it as unnecessary noise.